McPicker.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name             = 'McPicker'
-  s.version          = '1.0.0'
+  s.version          = '1.1.0'
```

Comment:
Per semantic versioning this isn't a new feature so it would be `1.0.1`. If this change is not reverse compatible with swift 4.1 then we need to jump to `2.0.0`

Reply:
Actually I was thinking not to make it reverse compatible, that would make the code cleaner without the compiler `#if` and the `#warning`, and we could jump to `2.0.0`. What do you think?

Reply:
Let's jump to `2.0.0` and could you please add `spec.swift_version = '4.2'` per https://github.com/kmcgill88/McPicker-iOS/issues/37